internals/testing/test-bundler.js

```diff
@@ -13,6 +13,7 @@ import Adapter from 'enzyme-adapter-react-16';
 Enzyme.configure({ adapter: new Adapter() });
 
 global.window.L = L;
+global.window.alert = msg => msg;
```

Comment:
Do we really need this? Just to test a request value?

Reply:
Yes we do; JSDOM doesn't have the `alert` method implemented.